meter/e3dc.go

```diff
@@ -87,6 +88,7 @@ func NewE3dc(cfg rscp.ClientConfig, usage templates.Usage, dischargeLimit uint32
 		usage:          usage,
 		conn:           conn,
 		dischargeLimit: dischargeLimit,
+		cfg:            cfg,
```

Comment:
Instead of storing the config I would suggest to store a function `retry` that disconnects and reconnects. The retry error- instead of being ignored- should be checked again.